tmdb-api/tmdb-assignment.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import pandas as pd
+import json
+
+api_key = "69e0d756f57f03c3427c9622298bc4dc"
+url = f"https://api.themoviedb.org/3/search/movie?api_key={api_key}"
+
+data = requests.get(url)
+
+starwars_url = url + "&query=Star%20Wars"
```

Comment:
There are multiple pages of data that you will have to use a parameter to access. You can get the total number of pages from the url used here.